shell/platform/darwin/common/framework/Source/FlutterCodecs.mm

```diff
@@ -73,7 +73,7 @@ - (NSData*)encode:(id)message {
 }
 
 - (id)decode:(NSData*)message {
-  if (message == nil)
+  if (message.length == 0)
```

Comment:
What's the reason for this change? It's actually changing the behavior of this API; before this change it's an assertion failure to send a non-nil but zero-length message, and now it's not.

(And if this is a change we want to make, then the now-redundant `if` below it should be removed.)

Reply:
I was getting an assertion failure since the data I was getting from the response was not null, but length 0. I now think that was WAI here, but added a length > 0 check in the channel to avoid hitting this assertion